pkg/webhook/managedresource/managedresource_validating_webhook.go

```diff
@@ -0,0 +1,99 @@
+/*
```

Comment:
Please update to use microsoft copyright comment for new file https://github.com/Azure/fleet/blob/main/cmd/crdinstaller/main.go#L1-L4